### PR TITLE
Post Launch feedback for projects facelift

### PR DESCRIPTION
**Pull request by mohitb35**
- Improve date formatting by adding string cleanup and zero-padding. 
- Simplify conditional rendering in the KeyInfo component for better readability and maintainability. 
- Correct minor alignment issues in the site dropdown and explore buttons.
- Allow project details to overflow till bottom of page
- Potential fix to make sure map resizes as per container dimensions
- Minor adjustment to position of zoom buttons on mobile (android/iOS) to avoid hiding behind footer
- Prevent map re-centering when the project filter/search result set is >30 projects long
- italicises scientific name in sample plant location details
- formats site area in site dropdown as per locale

### CODE REVIEW

File: src/features/projectsV2/ProjectsMap/SingleProjectView.tsx [hunk 1]
```diff
@@ -67,15 +67,22 @@ const SingleProjectView = ({ mapRef }: Props) => {
...
         // Zoom into the project location that has no site
-        zoomToLocation(setViewState, longitude, latitude, 10, 4000, mapRef);
+        zoomToLocation(
+          handleViewStateChange,
+          longitude,
+          latitude,
+          10,
+          4000,
+          mapRef
+        );
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/features/projectsV2/ProjectsMap/SingleProjectView.tsx:85]: Not relevant to this PR

**Review by mariahosfeld — CHANGES_REQUESTED** [review]
Two points:

- [x] the separator also needs to be applied on the list of planted species, we can have 4 digits here as well
image
- [ ] with the applied fix, the zoom out control is now totally overlapped by the footer on my mobile
image

**Review by mariahosfeld — COMMENTED** [review]
All functionalities work for me now.

**Review by mariahosfeld — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     ".next/cache"
   ],
   "scripts": {
+    "dev-https": "next dev --experimental-https",
```
Comment (sunilsabatp) [inline package.json:9]: Could you remove the `https `script ?

File: .gitignore [hunk 3]
```diff
@@ -37,6 +37,8 @@ cypress.env.json
 # certificates
 ./certificates/localhost.crt
 ./certificates/localhost.key
+certificates/localhost-key.pem
+certificates/localhost.pem
```
Comment (sunilsabatp) [inline .gitignore:41]: Remove this if it is not needed.

File: package.json (hunk 2 above)
Comment (mohitb35) [reply package.json:9]: This was added to make it easier to run https in dev mode

File: .gitignore (hunk 3 above)
Comment (mohitb35) [reply .gitignore:41]: This was added to make it easier to run https in dev mode, and prevent the locally generated certificates from being uploaded to the repo

**Review by sunilsabatp — APPROVED** [review]
LGTM

File: src/utils/countryCurrency/getFormattedDate.ts [hunk 4]
```diff
@@ -3,23 +3,30 @@
...
+      // Zero-padding month and day if needed
+      const [year, month, day] = cleanDateString.split('-');
+      const isoDateString = `${year}-${month.padStart(2, '0')}-${day.padStart(
+        2,
+        '0'
+      )}`;
+      return format(parseISO(isoDateString), 'LLLL d, yyyy', {
+        locale: localeMapForDate[localStorage.getItem('language') || 'en'],
+      });
+    } else {
```
[the remark this answers is not in the document]
Comment (mohitb35) [reply src/utils/countryCurrency/getFormattedDate.ts]: Resolved in https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2302#:~:text=for%20site%20area-,a6ed9e2,-coderabbitai

**Pull request merged**